asset/js/main.js

```diff
@@ -0,0 +1,29 @@
+document.addEventListener('DOMContentLoaded', () => {
+    loadHTML('/src/layouts/header.html', 'header-placeholder', '/asset/css/header.css');
+});
+
+function loadHTML(filename, elementId, cssFile = null) {
+    fetch(filename)
+        .then(response => response.text())
+        .then(data => {
+            document.getElementById(elementId).innerHTML = data;
+            if (cssFile) {
+                loadCSS(cssFile);
+            }
+
+            // Add languageSwitcher script
+            const script = document.createElement('script');
+            script.src = '/asset/js/languageSwitcher.js';
+            script.defer = true;
+            script.onload = setupLanguageSwitcher;
+            document.body.appendChild(script);
```

Comment:
```suggestion
            setupLanguageSwitcher();
```
Chỗ này chỉ cần sửa thành fuct setupLanguageSwitcher